tests/__init__.py

```diff
@@ -105,7 +105,7 @@
 VALID_VERIFY_SSL = True
 VALID_USER = "valid_user"
 VALID_USER_2SA = "valid_user_2sa"
-VALID_PASSWORD = "valid_password"
+VALID_PASSWORD = "valid_password"  # noqa: S105
```

Comment:
why some constant are in this file and not in const.py ?

Reply:
Probably because they are used only in this file.

I've look at the const.py file. I even think we can split it in two with one part related to the error only.

But that's not the purpose of this pr.